components/director/internal/open_resource_discovery/service.go

```diff
@@ -95,19 +90,13 @@ func (s *Service) SyncORDDocuments(ctx context.Context) error {
 		globalResourcesOrdIDs = make(map[string]bool)
 	}
 
-	pageCount := 1
-	pageSize := 200
-
-	pageCursor := ""
-	hasNextPage := true
-
-	queue := make(chan *model.Application)
-	appErrors := int32(0)
+	queue := make(chan *model.Webhook)
+	var webhookErrors = int32(0)
 
 	wg := &sync.WaitGroup{}
 	wg.Add(s.config.maxParallelApplicationProcessors)
```

Comment:
Rename this configuration (here and in YAML files) from `maxParallelApplicationProcessors` to `maxParallelWebhookProcessors` because it essentially becomes that.